config/locales/zh.yml

```diff
@@ -1132,15 +1132,15 @@ idv.messages.gpo.start_over_html: 如果该地址不对，你需要%{start_over_
 idv.messages.gpo.start_over_link_text: 重新开始并用你新地址进行验证。
 idv.messages.gpo.timeframe_html: 你会在<strong> 5 到 10 天</strong> 里收到带有<strong>验证码</strong> 的信。
 idv.messages.otp_delivery_method_description: 如果你在上面输入的是座机电话，请在下边选择“接听电话”。
-idv.messages.phone.alert_html: '<strong>输入一个这样的电话号码：</strong>'
-idv.messages.phone.description: 我们会将该号码与记录核对并给你发个一次性代码。这是为了帮助你验证身份。
+idv.messages.phone.alert_html: '<strong>输入一个符合以下条件的电话号码：</strong>'
```

Comment:
`alert_html` is the "Enter a phone number that is:" prompt. It didn't change in most translations, but the Chinese version did.